public/css/app.css

```diff
@@ -669,6 +669,10 @@ div.sprite {
     margin: auto;
   }
 
+  .mw-860 {
```

Comment:
Suspicious @pdehaan is suspicious.
Not sure if this should be `.mw-600 {..}` or if I'm missing larger picture naming schemes.
But I also suspect this would override the `max-width` of the shadowed `.mw-860` selector above on L161-163.

Reply:
Confirmed that this does overwrite the first instance of `.mw-860`. 

@lesleyjanenorton It may be easier to read to include break-point specific utility classes. Example would be making this instance an entirely new class of `md-mw-860`, the `md` for Medium devices for all devices with a max-screen size of 800px. 

This convention may require more updates for other instances, but would make it extremely readable how its supposed to have.